client/src/App.css

```diff
@@ -0,0 +1,452 @@
+    font-family: Raleway, sans-serif;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    -webkit-transform: translate(-50%, -50%);
+    -ms-transform: translate(-50%, -50%);
+    transform: translate(-50%, -50%);
+    text-align: center;
+    background-color: #2978a0;
+  }
+
+#name{
+    color: #bcab79;
+    background-color: #2978a0;
+    font-weight: 600;
+}
+
+/* Header CSS---------------------------------------------------- */
+#button {
+    background-color: #315659; 
+    border: none;
+    font-family: 'Raleway', sans-serif;
+    font-size: 18px;
+    color: #c6e0ff;
+    padding: 10px 25px;
+    display: inline-block;
+    cursor: pointer;
+    text-decoration: none;
+    margin: 10px;
+    border-radius: 10px;
+    margin-top: 50px;
+  }
+  #button:hover {
+    background-color: #253031;
+    opacity: 50%;
+    transition: all .5s ease-out;
+    color: #2978a0;
+  }
+
+  #header-buttons{
+      display: flex;
+      justify-content: flex-end;
+      margin-bottom: 20px;
+  }
+  #logo-photo{
+    float: left;
+    margin-top: 20px;
+    padding-left: 15px;    
+  }
+
+  /* Portfolio CSS---------------------------------------------- */
+
+  .portfolio{
+    display: grid;
+    grid-gap: 3rem;
+    grid-template-columns: 5% 35% 15% 35% 10%;
+    grid-template-rows: 10% 30% 30% 30%;
+    grid-template-areas: 
+    ". title title title ."
+    ". geovermont . yelp ."
+    ". remock . tictactoe ."
+    ". . zork . ."   
+                   
+}
+
+.portfolio h1{
+    font-size: 50px;
+    color: #c6e0ff;
+    font-weight: 700;
+}
+
+.portfolio h2{
+    margin-top: -10px;
+    font-size: 18px;
+    color: #c6e0ff;
+}
+.portfolio h3{
+    font-size: 28px;
+    color: #bcab79;
+    font-family: Libre Baskerville, sans-serif;
+}
+
+.portfolio h4{
+    font-size: 16px;
+    color: #c6e0ff;
+}
+
+#title{
+    grid-area: title;
+    font-family: Raleway, sans-serif;
+    justify-content: center;
+    text-align: center;
+    margin-top: 50px;
+    
+}
+#geovermont{
+    grid-area: geovermont;
+    text-align: center;
+}
+#yelp{
+    grid-area: yelp;
+    text-align: center;
+    
+}
+#remock{
+    grid-area: remock;
+    text-align: center;  
+}
+#tictactoe{
+    grid-area: tictactoe;
+    text-align: center;
+}
+#zork{
+    grid-area: zork;
+    text-align: center;
+    padding: 0;
+    place-self: center;
+    margin-bottom: 200px;
+}
+.portfolio img{
+    text-align: center;
+    border: 1px solid #ddd;
+    border-radius: 10px;
+    padding: 5px;
+    width: 500px;
+    height: 350px;
+}
+
+.portfolio #zork img{
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+/* Footer CSS------------------------------------------------------- */
+
+.footer{
+    display: flex;
+    justify-content: space-evenly;
+    background-color: #253031;
+    text-align: center;
+    align-content: center;
+    flex-shrink: 0;
+    margin-top: auto;
+    padding: 20px;
+    width: 100%;
+    box-shadow: 0 50vh 0 50vh #253031;
+}
```

Comment:
Your footer css is causing a bottom scroll bar to appear on the screen because the width of your footer is bigger than the screen. You can fix this a couple different ways:

One is to remove the width rule. Because divs are block level elements, they will take up the full width of the screen by default. Since you have added a 20px padding around the whole thing, it makes the width of the div the full width of the screen, plus 20px on each side, causing that scroll bar.

Another way is to remove the padding you've added. But, if you want a 20px padding, and you want the div to stretch the full width and no more, then you can add the rule ```box-sizing: border-box```. Here is a link to the [documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing). This essentially restricts all borders and padding added to be added on the inside of the edges of the box, which will keep the size of the box the same, rather than adding them to the outside and causing the box to get bigger.